railties/lib/rails/generators/rails/plugin/templates/app/helpers/%namespaced_name%/application_helper.rb.tt

```diff
@@ -1,5 +1,18 @@
+<% if skip_webpack? -%>
 <%= wrap_in_modules <<~rb
   module ApplicationHelper
   end
 rb
 %>
+<% else -%>
+<%= wrap_in_modules <<~rb
+  module ApplicationHelper
+    include ::Webpacker::Helper
+    \n
```

Comment:
What is the line?

Reply:
My bad. That is not needed. I'll remove it.